functional/config.rb

```diff
@@ -0,0 +1,58 @@
+# Size of the CoreOS cluster created by Vagrant
```

Comment:
No, we can not. Because in this case we will have to create a hook which will wait until provisioning has been finished. And we don't control cloudinit execution in vagrant. This will complicate things.